ci/rt.sh

```diff
@@ -0,0 +1,255 @@
+#!/bin/bash
+######################################################################
+# This script is desined for UPP regression tests run by UPP developer.
+# Wen Meng, 12/2020, First version.
+# Fernando Andrade-Maldonado 5/2023 rework for CLI Options
+# Fernando Andrade-Maldonado / Wen Meng 9/2023 Add Hercules, fix typos, and refactor
+#
+######################################################################
+
+git_branch="develop"
+git_url="https://github.com/NOAA-EMC/UPP.git"
+clone_on="no"
+
+while getopts a:w:h:r:t:b:u:c opt; do
+  case $opt in
+    a) accnr=${OPTARG}
+        ;;
+    w) workdir=${OPTARG}
+        ;;
+    h) homedir=${OPTARG}
+        ;;
+    r) rundir=${OPTARG}
+        ;;
+    t) test_v=${OPTARG}
+        ;;
+    b) git_branch=${OPTARG}
+        ;;
+    u) git_url=${OPTARG}
+        ;;
+    c) clone_on="yes"
+	;;
+  esac
+done
+
+#UPP working copy
+test_v=${test_v:-`pwd`/../}
+if [[ $clone_on == "yes" ]]; then
+  rm -rf $test_v
+  mkdir -p $test_v
+  git clone -b $git_branch $git_url $test_v
+fi
+export svndir=${test_v}
+
+#Assume a nems account to run with
+accnr=${accnr:-"nems"}
+
+#Build UPP executable
+build_exe=yes
+
+#Choose run specific model
+run_nmmb=yes
+run_gfs=yes
+run_gefs=yes
+run_fv3r=yes
+run_rap=yes
+run_hrrr=yes
+run_hafs=yes
+run_rtma=yes
+
+#find machine
+mac=$(hostname | cut -c1-1)
+mac2=$(hostname | cut -c1-2)
+mac3=$(hostname | cut -c1-4)
+if [ $mac2 = hf ]; then # for HERA
+ export machine=HERA
+ export homedir=${homedir:-"/scratch2/NAGAPE/epic/UPP/test_suite"}
+ export rundir=${rundir:-"/scratch1/NCEPDEV/stmp2/${USER}"}
+ module use /scratch1/NCEPDEV/nems/role.epic/hpc-stack/libs/intel-2022.1.2/modulefiles/stack
+ module load hpc/1.2.0
+ module load hpc-intel/2022.1.2
+ module load hpc-impi/2022.1.2
+ module load prod_util
+elif [ $mac = O ] ; then
+ export machine=ORION
+ export homedir=${homedir:-"/work/noaa/epic/UPP"}
+ export rundir=${rundir:-"/work2/noaa/stmp/$USER"}
+ module use /work/noaa/epic-ps/role-epic-ps/hpc-stack/libs/intel-2022.1.2/modulefiles/stack
+ module load hpc/1.2.0
+ module load hpc-intel/2022.1.2
+ module load hpc-impi/2022.1.2
+ module load prod_util/1.2.2
+elif [ $mac3 = herc ] ; then
+ export machine=HERCULES
+ export homedir=${homedir:-"/work/noaa/epic/UPP"}
+ export rundir=${rundir:-"/work2/noaa/stmp/$USER"}
+ module use /work/noaa/epic/role-epic/spack-stack/hercules/spack-stack-dev-20230717/envs/unified-env/install/modulefiles/Core
+ module load stack-intel/2021.9.0
+ module load stack-intel-oneapi-mpi/2021.9.0
+ module load jasper/2.0.32
+ module load prod-util/1.2.2
+elif [ $mac = d -o $mac = c ]; then # WCOSS2
```

Comment:
@FernandoAndrade-NOAA Forgot to mention that WCOSS2 is not using slurm job scheduler. So rt.sh failed on WCOSS2. You may remove this block. We working on that later.

Reply:
Sounds good, I'll remove the WCOSS2 lines in the meantime then, thanks for testing that.